tests/integration.rs

```diff
@@ -329,6 +329,39 @@ fn test_get_current_user() -> Result<()> {
     Ok(())
 }
 
+#[test]
+#[cfg(feature = "min-redis-compatibility-version-7-4")]
+fn test_set_acl_categories() -> Result<()> {
+    let port: u16 = 6490;
+    let _guards = vec![start_redis_server_with_module("acl", port)
+        .with_context(|| "failed to start redis server")?];
+    let mut con =
+        get_redis_connection(port).with_context(|| "failed to connect to redis server")?;
+
+    let res: Vec<String> = redis::cmd("ACL").arg("CAT").query(&mut con)?;
+    assert!(res.contains(&"acl".to_owned()));
+
+    Ok(())
+}
+
+#[test]
+#[cfg(feature = "min-redis-compatibility-version-8-0")]
+fn test_set_acl_categories_commands() -> Result<()> {
+    let port: u16 = 6490;
+    let _guards = vec![start_redis_server_with_module("acl", port)
+        .with_context(|| "failed to start redis server")?];
+    let mut con =
+        get_redis_connection(port).with_context(|| "failed to connect to redis server")?;
```

Comment:
Haven't we merged the changes of mine where for all of this we just have a single function instead? Somewhere in `tests/utils.rs` or something?

Reply:
https://github.com/RedisLabsModules/redismodule-rs/pull/376/files here it is @MeirShpilraien please merge it, you approved it the last time and the freebsd failed, I guess, only due to their repositories having been shut down, or it got stuck, or something else which was usual for it to break.

Reply:
i do not see it there.

Reply:
Yes, we need to merge the PR for these changes to arrive. False alarm for now, but if we merge it before this one, then it would be nice to use it here as well.

Reply:
We have just merged the PR: https://github.com/RedisLabsModules/redismodule-rs/pull/376

You can use the function there to simplify the test code for the new tests.